### ISSUE TITLE
Socket failure when proxy enabled on Windows

### ISSUE THREAD

**Issue by EstevanTH** [EstevanTH/gpsd_javascript_relay#2]
Hi.

On Windows 10 (others not tested yet), if you enable a proxy server in your *Internet Options*, GPSD clients cannot start. They cannot get out of the "Stopped" state. The proxy server was inaccessible at that time, but I am not sure that this makes a difference.

![Proxy preview](http://puu.sh/zzeuh/320796c892.png)

This is a concern for me as use a proxy server when I am not in my vehicle.

**Comment by EstevanTH** [comment]
I have run the debugger on Windows 8.1, the proxy is enabled and valid.
The *QAbstractSocket* class fails while binding or connecting (if not explicitly bound), with the HTTP server as well.

There is nothing to do except avoiding *QAbstractSocket*. I am sorry.

**Comment by EstevanTH** [comment]
I did not notice that! Qt network classes use the default proxy server by default, even for a TCP server. That is pretty counter-intuitive, but I think that the fix is actually simple.
```cpp
QTcpServer::setProxy( QNetworkProxy::NoProxy )
QAbstractSocket::setProxy( QNetworkProxy::NoProxy )
```

**Comment by EstevanTH** [comment]
It is fixed in update 0.9.3a.
